submitted_models/bosdyn_spot/launch/common.rb

```diff
@@ -91,7 +91,7 @@ def _spawner(_name, _modelURI, _worldName, _x, _y, _z, _roll, _pitch, _yaw, _add
         <capacity>78.4</capacity>
         <resistance>0.061523</resistance>
         <smooth_current_tau>1.9499</smooth_current_tau>
-        <power_load>4.95</power_load>
+        <power_load>6.6</power_load>
```

Comment:
The `power_load` should be 9.9.  My apologies for giving you the incorrect value.  Your adjustment on your other PR caught this.

Reply:
Fixed.